### ISSUE TITLE
City Demo throwing errors

### ISSUE THREAD

**Issue by thefenry** [dustyroom-studio/quibli-doc#31]
**Describe the bug**
When using the latest version of Quibli 2.5.8 with unity 22.3.45f1 and you try running it the scene throws some errors. So far it seems to be isolated to this scene. 
I have removed the package, redownloaded the package. Restarted Unity but still nothing.

**To Reproduce**
1. Go to '[Demo] City'
2. Click on 'Run from editor'
3. Scroll down to 'If you have Error pause on console, it will open the console, otherwise navigate to the console'
4. Errors will appear

**Expected behavior**
No errors 

**Screenshots**
![image](https://github.com/user-attachments/assets/0e2227d7-bb6d-40c8-9261-1a6abc4b8640)

**Unity details**:
Quibli version 2.5.8
Unity 2022.3.45f1
Dev platform: WindowsEditor
Target platform: StandaloneWindows64
Render pipeline: 
URP installed: True, version 14.0.11
Render pipeline asset: Quibli URP Config
Renderers: Quibli Renderer, [Quibli] City Demo Scene Renderer, [Quibli] Sample Scene With Quibli Renderer, [Quibli] Nature Demo Scene Renderer
Rendering path: Forward
Color space: Linear
Quality config: Quibli URP Config
Graphics config: Quibli URP Config

**Additional context**

**Comment by dustyroom-studio** [comment]
Thank you for reporting this. We've seen other people also having this problem with Quibli, but so far we can't reproduce it no matter what we try.

Does your project have any other assets by any chance?

**Comment by dustyroom-studio** [comment]
I see people mention on forums that it may be due to the objects being static. Can you please try selecting the `Wires` GameObject and unchecking `Static` (including children)?
<img width="337" alt="image" src="https://github.com/user-attachments/assets/b56aa2b9-3b3b-4030-8155-158b49acf82a" />

**Comment by thefenry** [comment]
yes that seems to be the issue. Making that Gameobject and its children non static fixes it
